ocaml/xapi/quicktest_cbt.ml

```diff
@@ -14,20 +14,20 @@
 
 open Client
 open Quicktest_common
-exception Test_assertion_failed of string
+exception Test_failed of string
 
 let start session_id =
   let open Client in
 
   (* Helper for test failure due to unexpected error *)
   let report_failure error test_name test =
-    failed test (Printf.sprintf "%s failed: %s" test_name
-                   (ExnHelper.string_of_exn error)) in
+    raise Test_failed (Printf.sprintf "%s failed: %s" test_name
```

Comment:
Apparently we need more brackets here :disappointed: : `(Test_failed  (Printf.... ))`

Reply:
I've changed this back to failed test , because it's called when a failing exception is caught, and otherwise I was matching an exception with an exception that then wanted to be matched to another exception and I'm getting a headache just typing this so you can surely understand my reasoning.